### PR TITLE
Feat(workflow): support to send dingDing message

### PR DESCRIPTION
**Pull request by leejanee**
use case as follow:
```
notify: op.#DingDing & {
            token: "xxx"
            message: msgtype: "link"
            message: link: {
                text:       "Welcome to KubeVela Workflow"
                title:      "workflow"
                picUrl: "https://raw.githubusercontent.com/oam-dev/kubevela.io/main/docs/resources/KubeVela-03.png"
                messageUrl:     "https://github.com/oam-dev/kubevela"
            }
}
```

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes (https://github.com/oam-dev/kubevela/pull/2137)

**Special notes for your reviewer**:

### CODE REVIEW

File: pkg/stdlib/op.go [hunk 1]
```diff
@@ -107,6 +110,20 @@ import ("encoding/yaml")
   ] @step(2)
 }
 
+#DingDing: #Steps & {
```
Comment (hongchaodeng) [inline pkg/stdlib/op.go:113]: 1. The English name is **DingTalk**
2. Could we separate this kind of operation into another library path? Like `notification.Dingding`

**Review by hongchaodeng — COMMENTED** [review]
Have some naming concerns

File: pkg/stdlib/op.go (hunk 1 above)
Comment (leejanee) [reply pkg/stdlib/op.go:113]: >1.The English name is DingTalk
ok, I will fix it
> 2.Could we separate this kind of operation into another library path? Like notification.Dingding
目前workflow对于依赖管理还不完善,只能放在op这个package下

Comment (hongchaodeng) [reply pkg/stdlib/op.go:113]: > 目前workflow对于依赖管理还不完善,只能放在op这个package下

OK.
或者代码层面上能先分到两个文件吗？op.go 和 notification.go

Comment (leejanee) [reply pkg/stdlib/op.go]: > 或者代码层面上能先分到两个文件吗？op.go 和 notification.go
ok

**Review by hongchaodeng — APPROVED** [review]
lgtm

File: pkg/stdlib/providerLib.go [hunk 2]
```diff
@@ -95,6 +95,55 @@ http: #Do: {
...
+        feedCard?: *null | {
+                links: *null | [...*null | {
+                        text?:       string
+                        title?:      string
+                        messageUrl?: string
+                        picUrl?:     string
+                }]
+        }
+    }
+}
```
Comment (wonderflow) [inline pkg/stdlib/providerLib.go:146]: I suggest we use https://pkg.go.dev/embed for all these CUE files

**Pull request merged**